tests/unit/routes/index-test.js

```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
```

Comment:
Generally routes get tested within an acceptance test - so explicit route-tests can be removed.